core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -25,12 +25,20 @@ import org.apache.spark.ui.{UIUtils, WebUIPage}
 
 private[ui] class EnvironmentPage(parent: EnvironmentTab) extends WebUIPage("") {
   private val listener = parent.listener
-
+  val passwordProperties: Set[String] = Set("spark.ssl.keyPassword",
+    "spark.ssl.keyStorePassword", "spark.ssl.trustStorePassword")
+
+  def removePass(kv: (String, String)): (String, String) = {
```

Comment:
`private`? agree with filtering anything whose key has "password" (not case sensitive). Check for other instances where listener.sparkProperties is accessed too?